downstream/modules/hub/proc-obtain-images.adoc

```diff
@@ -58,6 +58,6 @@ $ podman images
 [role="_additional-resources"]
 .Additional resources
 
-* See link:redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting {ExecEnvShort}s.
+* See link:https://redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting {ExecEnvShort}s.
```

Comment:
Was removing the s a global change? If so, you missed this instance.

Reply:
Good catch!